content/articles/line-detection-using-hough-transform-in-matlab/index.md

```diff
@@ -0,0 +1,149 @@
+### Line detection using hough transform in Matlab
+### Table of content
+[Introduction](#introduction)
+[Prerequisites](#prerequisites)
+[Standard hough transform](#standard-hough-transform)
+[Using hough transform to detect a line](#using-hough-transform-to-detect-a-line)
+[Conclusion](#conclusion)
+
+### Introduction
+Hough transform is an algorithm that isolates specific shapes in an image. These desired features should are specified in a given parametric form. This algorithm is widely used in detecting regular curves. These curves are such as lines, curves and ellipses etc. However, the general application of hough transform is impossible to attain the simple analytical feature description. For example, line detection is an important feature that helps analyze two objects. It can give a more important relationship between the two objects.
```

Comment:
What desired features are you referring to here?

Hough transform is an algorithm that isolates specific shapes in an image. `These desired features should are specified in a given parametric form.` This algorithm is widely used in detecting regular curves. These curves are such as lines, curves and ellipses etc. However, the general application of hough transform is impossible to attain the simple analytical feature description. For example, line detection is an important feature that helps analyze two objects. It can give a more important relationship between the two objects.